### DISCUSSION TITLE
I am pretty curious, what exploits work for u {POST IN COMMENTS}

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1321] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
POST IN COMMENTS

**Comment by ghost** [comment]
And also how do u use sparkvue????

**Reply by FunnyCorgi** [reply]
@SIGMAMEWERSWILLBEMINEYEAH blobby-boi has a file for it on his github somewhere I believe
If not, I have I file I think might work for you(but try blobby-boi first, his is where I got mine from).

**Comment by ghost** [comment]
None, I have contentkeeper, brick might work but my computer is nissa board chrome 130 kernver 4

**Reply by thatboyo** [reply]
dude i have CK too. i hate it

**Reply by MasterKia** [reply]
@thatboyo Hello Shrek

**Comment by Sam-hacks1** [comment]
only exthang works for me

**Comment by Perplexedykwhat** [comment]
I can use rigtools, URun, ExtHang3r, and a lot more

**Reply by sadgimkitgamer** [reply]
what is Urun?

**Reply by RacoonyCoon** [reply]
Urun is the same thing as UBlock just allowing you to execute JavaScript if bookmarklets are blocked

**Comment by sites1235** [comment]
I am on v109 kernver 0 and unenrolled so i can use pretty much every exploit and my board is octopus

**Reply by SlitheringLizard1** [reply]
I have v111 with octopus and I am scared to unenroll

**Comment by Luigi-Master** [comment]
exthang3r and dextensify

**Reply by SlitheringLizard1** [reply]
same but dex is extremely inconsistent

**Comment by dudedecliedddddddd** [comment]
exthang3r but only for like 15 minutes>

**Reply by Sam-hacks1** [reply]
fax

**Reply by ghost** [reply]
me too

**Reply by WalkTheEarth** [reply]
let me guess, Lightspeed?

**Reply by sadgimkitgamer** [reply]
yes

**Comment by schoolexploitkid** [comment]
sparkvue, lego wedo, chrome://chrome-signin, new securly kill exploit, rigtools, exthanger, dextensify, maybe caudns havent tried, maybe new killcurly method

**Reply by SlitheringLizard1** [reply]
what is chrome://chrome-signin exploit

**Comment by Dario-git-hub** [comment]
JPGMC works for me (i have iRiS school installed)

**Comment by WalkTheEarth** [comment]
exthang3r. Dextensify works every now and then.

**Comment by HelloIloveGimkit2** [comment]
[br1ck](https://br1ck.vercel.app)

**Comment by ghost** [comment]
cool guys chat y'all actually can do more than me

**Comment by jazzymakesstuff** [comment]
ext hang3r

**Comment by ghost** [comment]
[an unenroll exploit](https://github.com/coder0-code/dunroll)

**Reply by SlitheringLizard1** [reply]
how do I do the rollback
